src/store/modules/user/actions.ts

```diff
@@ -81,7 +81,15 @@ const actions: ActionTree<UserState, RootState> = {
       commit(types.USER_INFO_UPDATED, current);
       showToast(translate("Time zone updated successfully"));
     }
+  },
+
+  /**
+   * Set User Instance Url
+   */
+  setUserInstanceUrl ( { state, commit }, payload){
+    commit(types.USER_INSTANCE_URL,payload)
   }
+
```

Comment:
Remove extra lines

Reply:
Removed the unwanted lines.
Thank You for reviewing my code.